### ISSUE TITLE
Craft 4: Element query executed before Craft is fully initialized. (initBundles function)

### ISSUE THREAD

**Issue by icreatestuff** [solspace/craft3-freeform#720]
**Describe the bug or issue you're experiencing**

Similar to what is described [here](https://github.com/solspace/craft-freeform/issues/582) with log files being filled up with this warning related to Freeform.
However, it seems to be from a different area of the plugin initialisation.

```
2023-06-29 07:46:54 [web.WARNING] [craft\elements\db\ElementQuery::prepare] Element query executed before Craft is fully initialized. Stack trace: 
#0 /htdocs/vendor/yiisoft/yii2/db/QueryBuilder.php(227): craft\elements\db\ElementQuery->prepare() 
#1 /htdocs/vendor/yiisoft/yii2/db/Query.php(157): yii\db\QueryBuilder->build() 
#2 /htdocs/vendor/yiisoft/yii2/db/Query.php(287): yii\db\Query->createCommand() 
#3 /htdocs/vendor/craftcms/cms/src/db/Query.php(275): yii\db\Query->one() 
#4 /htdocs/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1590): craft\db\Query->one() 
#5 /htdocs/vendor/craftcms/cms/src/elements/User.php(519): craft\elements\db\ElementQuery->one() 
#6 /htdocs/vendor/yiisoft/yii2/web/User.php(698): craft\elements\User::findIdentity() 
#7 /htdocs/vendor/craftcms/cms/src/web/User.php(500): yii\web\User->renewAuthStatus() 
#8 /htdocs/vendor/yiisoft/yii2/web/User.php(199): craft\web\User->renewAuthStatus() 
#9 /htdocs/vendor/yiisoft/yii2/web/User.php(377): yii\web\User->getIdentity() 
#10/htdocs/vendor/yiisoft/yii2/base/Component.php(139): yii\web\User->getId() 
#11/htdocs/vendor/solspace/craft-freeform/packages/plugin/src/Bundles/Captchas/HCaptcha.php(28): yii\base\Component->__get() 
#12 Solspace\Freeform\Bundles\Captchas\HCaptcha->__construct() 
#13/htdocs/vendor/solspace/craft-freeform/packages/plugin/src/Freeform.php(978): ReflectionClass->newInstance() 
#14/htdocs/vendor/solspace/craft-freeform/packages/plugin/src/Freeform.php(303): Solspace\Freeform\Freeform->initBundles() 
#15/htdocs/vendor/yiisoft/yii2/base/BaseObject.php(109): Solspace\Freeform\Freeform->init() 
#16/htdocs/vendor/yiisoft/yii2/base/Module.php(161): yii\base\BaseObject->__construct() 
#17/htdocs/vendor/craftcms/cms/src/base/Plugin.php(122): yii\base\Module->__construct() 
#18 craft\base\Plugin->__construct() 
#19/htdocs/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs() 
#20/htdocs/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build() 
#21/htdocs/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get() 
#22/htdocs/vendor/craftcms/cms/src/Craft.php(54): yii\BaseYii::createObject() 
#23/htdocs/vendor/craftcms/cms/src/services/Plugins.php(943): Craft::createObject() 
#24/htdocs/vendor/craftcms/cms/src/services/Plugins.php(228): craft\services\Plugins->createPlugin() 
#25/htdocs/vendor/craftcms/cms/src/base/ApplicationTrait.php(1580): craft\services\Plugins->loadPlugins() 
#26/htdocs/vendor/craftcms/cms/src/web/Application.php(108): craft\web\Application->_postInit() 
#27/htdocs/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\web\Application->init() 
#28/htdocs/vendor/yiisoft/yii2/base/Application.php(204): yii\base\BaseObject->__construct() 
#29 yii\base\Application->__construct() 
#30/htdocs/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs() 
#31/htdocs/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build() 
#32/htdocs/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get() 
#33/htdocs/vendor/craftcms/cms/src/Craft.php(54): yii\BaseYii::createObject() 
#34/htdocs/vendor/craftcms/cms/bootstrap/bootstrap.php(256): Craft::createObject() 
#35/htdocs/vendor/craftcms/cms/bootstrap/web.php(40): require() 
#36/htdocs/web/index.php(11)
```

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 4.4.14
- Freeform Version: 4.1.3
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Comment by kjmartens** [comment]
Sorry about this @icreatestuff.

We'll check into this shortly. 🙂

**Comment by kjmartens** [comment]
This should now be resolved in Freeform 4.1.4 🙂
